app/src/main/java/com/github/se/eventradar/ui/login/SignUpScreen.kt

```diff
@@ -0,0 +1,469 @@
+                        } else {
+                            /* Nothing yet -> separation of the 2 PRs
+                            // User is already authenticated, store user data in Firestore
+                            CoroutineScope(Dispatchers.Main).launch {
+                                val imageUri = selectedImageUri.value
+                                    ?: Uri.parse("android.resource://" + context.packageName + "/" + R.drawable.placeholder)
+                                val userValues = hashMapOf(
+                                    "Name" to name.value,
+                                    "Surname" to surname.value,
+                                    "Phone Number" to phoneNumber.value,
+                                    "Birth Date" to birthDate.value,
+                                    "Email" to user.email,
+                                    "Profile Picture" to imageUri.toString()
+                                )
+
+                                // Add a new document with a generated ID into collection "users"
+                                Firebase.firestore.collection("users").add(userValues)
+                                    .addOnSuccessListener { documentReference ->
+                                        Log.d(
+                                            TAG,
+                                            "DocumentSnapshot added with ID: ${documentReference.id}"
+                                        )
+                                        // Navigate to the overview page
+                                        navigationActions.navController.navigate(Route.OVERVIEW)
+                                    }.addOnFailureListener { e ->
+                                        Log.w(TAG, "Error adding document", e)
+                                    }
+                            }
+
+                             */
+
+                            // Navigate to the overview page
+                            navigationActions.navController.navigate(Route.OVERVIEW)
+                        }
+                    },
+                    modifier = Modifier
+                        .wrapContentSize()
+                        .width(250.dp)
+                        .testTag("loginButton"),
+                    border = BorderStroke(width = 1.dp, color = MaterialTheme.colorScheme.primary),
+                    colors = ButtonDefaults.buttonColors(containerColor = MaterialTheme.colorScheme.primary),
+                ) {
+                    Image(
+                        painter = painterResource(id = R.drawable.logo),
+                        contentDescription = "Google Logo",
+                        modifier = Modifier
+                            .width(24.dp)
+                            .height(24.dp)
+                            .align(Alignment.CenterVertically),
+                    )
+                    Text(
+                        text = "Log In with Google",
```

Comment:
nit: (for next sprint) Not a coding problem, but I think that it would be better to replace the Button with "SIgn Up". Upon entering the app, I think that upon signing up, users can firstly Sign Up by Google (and once that's done will be asked to enter details for Sign Up). we can discuss with others on this though but I think it's a bit weird to have a Log In with Google in a Sign Up Page. @Viccha18 do you agree?

Reply:
I agree. Changed it!